src/emm/services/modules_service.py

```diff
@@ -231,3 +231,45 @@ def git_commit_modules(self, commit: str) -> None:
             module_data = self.get_module_data(module)
             module_location = Path(module_data.prefix) / module
             self.git_service.commit_all(commit, module_location)
+
+    def add_pr_link(self, comments: Dict[str, str]) -> None:
+        """
+        Add link to each module.
+
+        :param comments: Dictionary of module and its pull request url.
+        """
+        for module, pr_url in comments.items():
+            if module != "base":
+                module_data = self.get_module_data(module)
+                module_location = Path(module_data.prefix) / module
+                pr_links = "\n".join(
+                    "{} pr: {}".format(repo, link)
+                    for repo, link in comments.items()
+                    if repo != module
+                )
+                self.git_service.pr_comment(pr_url, pr_links, module_location)
+                print("module ", module, " pr: ", pr_url)
```

Comment:
We should avoid printing things out in the services layer and instead return data to the orchestrator to print out. That lets this code be reusable for other purposes that might not want the printing. For example, if someone wanted to extend this by using it as a library.